Time-Card/DRV/README.md

```diff
@@ -3,16 +3,34 @@
 Driver is based on a kernel module for CentOS and Ubunutu. 
 Kernel 5.2+ is recommended
 
+# Supported devices
+
+* Facebook's Time Card
+* Orolia's Atomic Reference Time Card (ART Card)
+
 ## Instruction
 
 Run the remake followed by modprobe ptp_ocp
 
-## Outcome
+## Time Card Outcome
 
 * After successfully loading the driver, you will see a new PTP POSIX clock, linking to the physical hardware clock (PHC) on the Time Card (`/dev/ptp*`). 
 * In addition there will be two uarts (`/dev/ttyS*`) and two i2c (`/dev/i2c-*`) devices added.
 * Now,you can use standard PTP4L tools such as phc2sys or ts2phc to copy, sync, tune, etc...
 
+## ART Card Outcome
```

Comment:
There should not be an difference between the two cards - the same API should be used.

There is /proc/driver/ocpN which collects the resources.